ci-operator/config/openshift/ansible-service-broker/openshift-ansible-service-broker-master.yaml

```diff
@@ -0,0 +1,12 @@
+build_root:
+  project_image_build:
```

Comment:
this should be `project_image`. It seems that we have a wrong name in the validator. I've opened [202](https://github.com/openshift/ci-operator/pull/202) to fix it.